src/main/resources/data/immersive_weathering/revelations/trees_cmy.json

```diff
@@ -0,0 +1,51 @@
+{
+  "advancement": "spectrum:milestones/reveal_colored_trees_cmy",
+  "block_states": {
+    "immersive_weathering:spectrum/orange_leaf_pile": "immersive_weathering:oak_leaf_pile",
+    "immersive_weathering:spectrum/magenta_leaf_pile": "immersive_weathering:oak_leaf_pile",
+    "immersive_weathering:spectrum/light_blue_leaf_pile": "immersive_weathering:oak_leaf_pile",
+    "immersive_weathering:spectrum/yellow_leaf_pile": "immersive_weathering:oak_leaf_pile",
+    "immersive_weathering:spectrum/lime_leaf_pile": "immersive_weathering:oak_leaf_pile",
+    "immersive_weathering:spectrum/pink_leaf_pile": "immersive_weathering:oak_leaf_pile",
+    "immersive_weathering:spectrum/cyan_leaf_pile": "immersive_weathering:oak_leaf_pile",
+    "immersive_weathering:spectrum/purple_leaf_pile": "immersive_weathering:oak_leaf_pile",
+    "immersive_weathering:spectrum/blue_leaf_pile": "immersive_weathering:oak_leaf_pile",
+    "immersive_weathering:spectrum/green_leaf_pile": "immersive_weathering:oak_leaf_pile",
+    "immersive_weathering:spectrum/red_leaf_pile": "immersive_weathering:oak_leaf_pile"
+  },
+  "items": {
+    "immersive_weathering:spectrum/orange_bark": "immersive_weathering:oak_bark",
+    "immersive_weathering:spectrum/magenta_bark": "immersive_weathering:oak_bark",
+    "immersive_weathering:spectrum/light_blue_bark": "immersive_weathering:oak_bark",
+    "immersive_weathering:spectrum/yellow_bark": "immersive_weathering:oak_bark",
+    "immersive_weathering:spectrum/lime_bark": "immersive_weathering:oak_bark",
+    "immersive_weathering:spectrum/pink_bark": "immersive_weathering:oak_bark",
+    "immersive_weathering:spectrum/cyan_bark": "immersive_weathering:oak_bark",
+    "immersive_weathering:spectrum/purple_bark": "immersive_weathering:oak_bark",
+    "immersive_weathering:spectrum/blue_bark": "immersive_weathering:oak_bark",
+    "immersive_weathering:spectrum/green_bark": "immersive_weathering:oak_bark",
+    "immersive_weathering:spectrum/red_bark": "immersive_weathering:oak_bark"
+
+  },
+  "item_name_replacements": {
+    "immersive_weathering:spectrum/orange_bark": "Oak Bark?",
+    "immersive_weathering:spectrum/magenta_bark": "Oak Bark?",
+    "immersive_weathering:spectrum/light_blue_bark": "Oak Bark?",
+    "immersive_weathering:spectrum/yellow_bark": "Oak Bark?",
+    "immersive_weathering:spectrum/lime_bark": "Oak Bark?",
+    "immersive_weathering:spectrum/pink_bark": "Oak Bark?",
+    "immersive_weathering:spectrum/cyan_bark": "Oak Bark?",
+    "immersive_weathering:spectrum/purple_bark": "Oak Bark?",
+    "immersive_weathering:spectrum/blue_bark": "Oak Bark?",
+    "immersive_weathering:spectrum/green_bark": "Oak Bark?",
+    "immersive_weathering:spectrum/red_bark": "Oak Bark?"
```

Comment:
The other spectrum blocks do use scrambled names instead.
Why should we handle it differently here?

Reply:
Unlike most spectrum blocks, this is something that a player is VERY likely to encounter in natural gameplay, as stripping a colored tree will yield its respective bark regardless of progression. Additionally, my planned cloak overhaul PR (that is waiting on changes to Revelationary) will be adding more instances like this.